ui/Screen/Funding/Pool/Outgoing.svelte

```diff
@@ -5,14 +5,22 @@
   import Support from "./Outgoing/Support.svelte";
 
   import * as fundingPool from "../../../src/funding/pool";
+  import * as transaction from "../../../src/transaction";
 
   export let pool: fundingPool.Pool;
+
+  let ongoingOnboardingTx = false;
+  transaction.store.subscribe(_ => {
+    ongoingOnboardingTx = transaction.ongoing(
+      transaction.TxKind.SupportOnboarding
+    );
+  });
 </script>
 
 <Remote store={pool.data} let:data={poolData}>
   {#if fundingPool.isOnboarded(poolData)}
     <Support bind:pool />
   {:else}
-    <GetStarted bind:pool />
+    <GetStarted ongoingTx={ongoingOnboardingTx} bind:pool />
```

Comment:
Meaning, once the user approved the transaction onboarding their support plan, we need to wait for the transaction to be included in a block. Until then, we show the `GetStarted` screen but its button will now be disabled, showing that it is waiting for the tx.